### ISSUE TITLE
Cambio en producto 5

### ISSUE THREAD

**Issue by AlvaroHumeres** [MinCiencia/Datos-COVID19#313]
Hola! no sé si es un error o cambiaron algo en el producto 5:

https://github.com/MinCiencia/Datos-COVID19/blob/master/output/producto5/TotalesNacionales.csv

Pero los casos totales pasaron de 127k a 13k

Gracias!

**Comment by pabloalarconsoto** [comment]
Creo que el algoritmo que lee el reporte no consideró la última cifra, por ende debería haber sido algo así como 130k

**Comment by fzmolina** [comment]
Gracias por sus reportes. fue corregido

**Comment by paarriagadap** [comment]
Otra cosa: sólo en un archivo de los 3 se cambió el número de muertes de 1637 a 2190. Sería bueno que todos queden con ese mismo dato.

**Comment by LuisAcunaSM** [comment]
Siguen los errores en este fichero, podría automatizarse su diagnóstico mediante detección de outliers, por ejemplo. A los valores _missing_ en `casos recuperados` desde el 2 de junio, hoy tenemos la cifra de 576 en `casos nuevos con síntomas`, 13.415 en `casos totales` y 645 en `casos nuevos con síntomas` me imagino que por error de parseo o conversión en el punto decimal. De antemano gracias por la corrección, el trabajo que están haciendo es vital para el enfrentamiento de esta catástrofe de abordaje de pandemia.

**Comment by dna33** [comment]
Hola @LuisAcunaSM, el 13415 que reportas ya fue corregido por @fzmolina dos horas atrás a raíz de lo reportado por @pabloalarconsoto y @AlvaroHumeres.

Mantenemos el issue abierto para evaluar lo que pide @paarriagadap.

Saludos

**Comment by LuisAcunaSM** [comment]
Hice el pull y, salvo lo de los `casos recuperados`, se corrigió... horrible la pendiente. Gracias por el trabajo de todos ustedes.

**Comment by fzmolina** [comment]
Sí, gracias @dna33 . El error fue corregido. Los casos recuperados no se ha reportado porque ese valor reportado ha sido deprecado por el Ministerio de Salud. @dna33  puede dar más detalles del trabajo que se está gestando para poder disponer de números de casos recuperados a través de la investigación epidemiológica

**Comment by dna33** [comment]
Si, ver https://github.com/MinCiencia/Datos-COVID19/issues/293#issuecomment-640246557

**Comment by MinCiencia** [comment]
Hola, hemos hecho el cambio en el producto 5 como fue anunciado, se mantendrán por 2 semanas las columnas antiguas, y se añadieron 4 nuevas columnas con los conteos por criterios de fecha de diagnóstico y por fecha de inicio de síntomas.

**Comment by paarriagadap** [comment]
Hola! Veo que aunque hay un proceso de actualización para el archivo, en realidad no se ha actualizado los últimos dos días

**Comment by scornejob** [comment]
> Hola! Veo que aunque hay un proceso de actualización para el archivo, en realidad no se ha actualizado los últimos dos días
Solucionado

**Comment by dna33** [comment]
Hola a todos, como avisamos hace un par de semanas, vamos a deprecar las columnas que no se han estado actualizando.

1. Activos
2. Recuperados

Mantendremos las series para

1. Activos por FIS y por FD
2. Recuperados por FIS y por FD

**Comment by danmate7** [comment]
Hola a todos, es posible que agreguen un _fis o fd a las nuevas columnas, ya que tienen los mismos nombres.
Gracias!!
